src/stories/containers/cu-about/cu-about-container.tsx

```diff
@@ -48,6 +49,28 @@ const CuAboutContainer = () => {
   const filteredCategories = useMemo(() => getArrayParam('filteredCategories', router.query), [router.query]);
   const searchText = useMemo(() => getStringParam('searchText', router.query), [router.query]);
 
+  const handleScroll = () => {
+    const element = document.getElementById('hidden-element');
+    if (element != null) {
+      console.log('window.scrollY', window.scrollY, window.pageYOffset);
+      const bound = element?.getBoundingClientRect();
+      console.log('bound', bound.top, bound.y);
+      if (bound && bound?.y < 276) {
+        setHiddenTextDescription(false);
+      } else {
+        setHiddenTextDescription(true);
+      }
+      console.log('Scrolling...');
+    }
+  };
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll, true);
+
+    // Remove the event listener
+    return () => {
+      window.removeEventListener('scroll', handleScroll, true);
+    };
+  }, []);
   const filteredData = useMemo(() =>
```

Comment:
If the feature is ready can we remove the console logs?

Reply:
done